### DISCUSSION TITLE
Button node sends old payload messages(?)

### DISCUSSION THREAD

**Discussion by migsun** [zachowj/hass-node-red#172] [Q&A]
I created a button node to start an automation from HA lovelace UI.
This works as designed until I change the output properties. After a new or changed output the node sends the already used one togther with the new values. If I change furthermore the node sends more and more messages. This only stops when I restart node red. Is this a feature and old payloads will be saved/send or simple a unintended behavior? Thx 
![nodered](https://user-images.githubusercontent.com/90647100/225979693-73747191-f4eb-443c-8d05-789a4d7cc9a2.jpg)

**Comment by zachowj** [comment]
https://github.com/zachowj/node-red-contrib-home-assistant-websocket/issues/881
